introduction.tex

```diff
@@ -99,31 +99,40 @@ \subsection{Deployment Timeline}
 	\begin{center}
 		\includegraphics[scale=0.7]{./images/eu_future/power_plot.png}
 	\end{center}
-	\caption{Timeseries of installed nuclear capacity in \gls{EU}.}
+	\caption{The timeseries of installed nuclear capacity in the EU are separated by \texttt{Region}s in \Cyclus.
+			 The sudden drops in capacity is caused by nuclear phaseout plans by nations like Germany and Belgium.
```

Comment:
sudden drops in capacity ARE caused... I'm stopping here. More grammar checking needed. I'll give you a checklist.